__tests__/app.test.js

```diff
@@ -179,6 +179,28 @@ describe("ENDPOINT TESTING", () => {
             );
           });
       });
+      test("404: Should return 404 error when article_id does not exist", () => {
+        const newVote = 1;
+        const vote = { inc_votes: newVote };
+        return request(app)
+          .patch("/api/articles/1000")
+          .send(vote)
+          .expect(404)
+          .then((response) => {
+            expect(response.body.msg).toBe("Page Not Found");
```

Comment:
I think "Page not found" is a little misleading as to what may have gone wrong

Reply:
- [ X ] changed to Resource not found